### PR TITLE
[Outlook] (spam reporting) Create snippet for the getAsFileAsync method

### PR DESCRIPTION
**Pull request by samantharamon**
Creates a code snippet for the `getAsFileAsync` method. This snippet will be added to the reference documentation in a separate PR (pending DT PR merge).

Related PRs:
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66028
- https://github.com/OfficeDev/office-js-docs-reference/pull/1634
- https://github.com/OfficeDev/office-js-docs-pr/pull/4102

### CODE REVIEW

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Make sure to do any snippet mapping before you merge

File: samples/outlook/99-preview-apis/get-base64-encoding.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            }
+
+            console.log(asyncResult.value);
+          });
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p class="ms-font-m">This sample shows how to get the Base64 encoding of a message in read mode.</p>
```
Comment (timwan10) [inline samples/outlook/99-preview-apis/get-base64-encoding.yaml:26]: This sample shows how to get the message exported in EML format, and encoded in Base64 (or something like that... should make sure that EML is included there)

File: samples/outlook/99-preview-apis/get-base64-encoding.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <p><b>Required mode</b>: Message Read</p>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="get-base64-encoding" class="ms-Button">
+            <span class="ms-Button-label">Get Base64 encoding</span>
```
Comment (timwan10) [inline samples/outlook/99-preview-apis/get-base64-encoding.yaml:33]: Get EML File

File: playlists-prod/outlook.yaml [hunk 3]
```diff
@@ -772,5 +772,14 @@
   rawUrl: >-
     https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/outlook/99-preview-apis/set-displayed-body-subject.yaml
   group: Preview APIs
+  api_set:
+    Mailbox: preview
+- id: outlook-get-base64-encoding
```
Comment (timwan10) [inline playlists-prod/outlook.yaml:777]: outlook-get-eml-encoding

**Review by timwan10 — APPROVED** [review]
I approved, but the titles/comments/documents should all reference EML format instead of Base64. The EML just happens to be encoded as Base64.

File: samples/outlook/99-preview-apis/get-base64-encoding.yaml (hunk 1 above)
Comment (samantharamon) [reply samples/outlook/99-preview-apis/get-base64-encoding.yaml]: Made the change here, as well as in the DT PR.

**Comment by samantharamon** [comment]
> Make sure to do any snippet mapping before you merge

Will complete snippet mapping in a separate PR.

**Pull request merged**